### PR TITLE
Update to PHPUnit 6

### PR DESCRIPTION
**Pull request by SenseException**
Update to PHPUnit 6 because it's shiny and of course https://twitter.com/shochdoerfer/status/829235488850587649

### CODE REVIEW

**Comment by shochdoerfer** [comment]
Thanks for your work. I will release version 0.8. today or tommorrow which basically is version 0.7 + the PSR-11 migration. Thus I would kindly ask you to rebase the PR then to make sure the PSR-11 changes make it into your PR.

**Comment by SenseException** [comment]
Done.

**Comment by shochdoerfer** [comment]
Thank you @SenseException for your first contribution to Disco!

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     "bitexpert/slf4psrlog": "^0.1.3"
   },
   "require-dev": {
-    "phpunit/php-code-coverage": "^4.0.1",
-    "phpunit/phpunit": "^5.6",
+    "phpunit/php-code-coverage": "^5.0.0",
```
Comment (sebastianbergmann) [inline composer.json:21]: Why the explicit dependency on php-code-coverage?

Comment (shochdoerfer) [reply composer.json:21]: Fair point, we might not need it any more. Did not realize. Thanks for pointing out ;)

**Pull request merged**